languages/elixir/reference/README.md

```diff
@@ -0,0 +1,145 @@
+# Elixir reference
+
+## Concepts
+
+Below are the concepts that have been identified in Elixir for use in Exercism.
+
+### Functional
+
+- [Anonymous functions](../../../reference/concepts/anonymous_functions.md)
+- [Higher-order functions](../../../reference/concepts/higher_order_functions.md)
+- [Immutability](../../../reference/concepts/immutability.md)
+- [Pattern matching](../../../reference/concepts/pattern_matching.md)
+- [Pipelines](../../../reference/concepts/pipelines.md)
+- [Recursion](../../../reference/concepts/recursion.md)
+- Tail Call Optimization
+
+### Platform-specific
+
+- BEAM VM
+- Processes
+- Agent Concurrency Model
+- Erlang Interoperation
+- String vs Charlist
+  - UTF8/16/32 encoding
+  - byte_size vs string length
+- Interactive Console (REPL)
+- elixir mix
+  - directory structure
+  - mix tasks
+- Naming conventions
+- Eager Computation
+- Lazy Computation
+- Naming Conventions
+- TODO: more
+
+### Modules
+
+- Naming
+- Compiling
+- Scripts (*.exs) vs Code (*.ex)
+- Attributes
+- Alias
+- Require
+- Import
+- Use
+- TODO: more
+
+### Functions
+
+- Named functions
+  - Multi-line syntax
+  - One-line syntax
+- [Default arguments](../../../reference/concepts/default_arguments.md)
+- Guards
+- Capture Syntax
+
+### General
+
+- [Arithmetic](../../../reference/concepts/arithmetic.md)
+- Basic Operators
+- Logical Operators
+- Comparison Operators
+  - Type Comparison Hierarchy
+- Control Structures
+- Literal Forms of data structures (lists, keyword lists, maps)
+- TODO: more
+
+### Basic Types
+
+- [Atom](../../../reference/types/symbol.md)
+- Numbers
+  - Integer
+    - Binary, Octal, Hex forms
+    - Codepoints
+      - `?` operator
+  - Float
+    - Scientific Notation
+- Binary
+  - String
+    - String Interpolation
+  - Special Forms
+- List
+  - Notation `[head | tail]` syntax
+  - Charlist
+  - iodata
+  - chardata
+  - List Comprehensions
+    - Generators
+    - Filters
+    - Into
+- Tuple
+- Function
+- Port
+- Reference
+- PID
+
+### Complex Types
```

Comment:
Also Streams here?

Reply:
Fixed in #505 

https://github.com/exercism/v3/blame/elixir-reference-changes/languages/elixir/reference/README.md#L107